src/test/java/org/broadinstitute/hellbender/utils/recalibration/covariates/BQSRCovariateListUnitTest.java

```diff
@@ -107,7 +110,7 @@ public void testGetCovariatesByParsedName() {
 
     @Test
     public void testCovariateInitialize() {
-        StandardCovariateList scl = makeCovariateList();
+        BQSRCovariateList scl = makeCovariateList();
```

Comment:
This test seems redundant and unnecessary, since it's calling test code which is called in many other places. If you want to keep it, remove the variable, since its not necessary:
```suggestion
        makeCovariateList();
```